src/App.css

```diff
@@ -1,39 +1,52 @@
-.App {
-  text-align: center;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.disflex {
+  display: flex;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.calculator {
+  flex-direction: column;
+  width: 25%;
+  height: 100%;
+  margin: 7% auto;
+  border: 1px solid;
+  font-size: 20px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+.solution {
+  width: 100%;
+  text-align: end;
+  padding: 3% 1%;
+  background: grey;
   color: white;
 }
 
-.App-link {
-  color: #61dafb;
+button {
+  height: 7vh;
+}
+
+.operators {
+  width: 75%;
+  display: grid;
+  grid-template-columns: auto auto auto;
+}
+
+.two-btn {
+  grid-column-start: 1;
+  grid-column-end: 3;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+.orange-btn {
+  flex-direction: column;
+  width: 25%;
+}
 
-  to {
-    transform: rotate(360deg);
-  }
+.orange-btn button {
+  background: orange;
+  width: 100%;
+  border: 1px solid;
 }
```

Comment:
[OPTIONAL] You can do the same for your orange buttons as well.

Before:
![image](https://user-images.githubusercontent.com/38088818/171231642-8a365c9d-c2fa-436c-a63d-15f571aa3346.png)

After:
![image](https://user-images.githubusercontent.com/38088818/171231218-40684a4f-7fdd-4901-93c2-af5950e3a022.png)

Reply:
Thank you. Done with this required change.